src/js/components/sharedComponents/autocomplete/SuggestionHolder.jsx

```diff
@@ -0,0 +1,59 @@
+/**
+ * Created by michaelbray on 1/27/17.
+ */
+
+import React from 'react';
+import Suggestion from './Suggestion';
+
+const propTypes = {
+    suggestions: React.PropTypes.array,
+    shown: React.PropTypes.string,
+    selectedIndex: React.PropTypes.number,
+    select: React.PropTypes.func
+};
+
+const defaultProps = {
+    suggestions: [],
+    shown: '',
+    selectedIndex: 0
+};
+
+export default class SuggestionHolder extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.hidden = true;
+    }
+
+    render() {
+        const suggestions = [];
+        if (this.props.suggestions.length > 0 && this.props.shown !== 'hidden') {
+            this.hidden = false;
+        }
+        else {
+            this.hidden = true;
+        }
+
+        // Ensure we're only showing 10 results at most
+        for (let i = 0; i < Math.min(this.props.suggestions.length, 10); i++) {
```

Comment:
Might make sense to make the max suggestion length a prop (with `defaultProps` set to 10), so autocomplete is more easily customized in the future.